### PR TITLE
[repo] Move Utkarsh to Emeritus

### PR DESCRIPTION
**Pull request by utpilla**
I'll be focusing more on OpenTelemetry Rust and related work. I'm stepping down from the Approver role as I would not have sufficient time to dedicate to this repo.

### CODE REVIEW

**Review by cijothomas — APPROVED** [review]
Thanks for all the amazing contributions to OTel .NET story!

**Comment by Kielek** [comment]
Utharsh, thank you!

File: README.md [hunk 1]
```diff
@@ -262,6 +261,7 @@ Maintainer/Approver/Triager](https://github.com/open-telemetry/community/blob/ma
 * [Reiley Yang](https://github.com/reyang)
 * [Robert Paj&#x105;k](https://github.com/pellared)
 * [Sergey Kanzhelev](https://github.com/SergeyKanzhelev)
+* [Utkarsh Umesan Pillai](https://github.com/utpilla), Microsoft
```
Comment (Kielek) [inline README.md:264]:
```suggestion
* [Utkarsh Umesan Pillai](https://github.com/utpilla)
```

**Review by Kielek — APPROVED** [review]

**Review by rajkumar-rangaraj — APPROVED** [review]
It's sad to see you go out of this repo. Thank you for your incredible efforts and contributions!

**Review by CodeBlanch — APPROVED** [review]
Thank you @utpilla for all the great contributions!

**Pull request merged**